components.json

```
@@ -12,7 +12,7 @@
   },
   "aliases": {
     "components": "@/components/shadcn",
-    "utils": "@/lib/shadcn/utils",
+    "utils": "@/lib/helpers",
```

I think this should be changed back 

```suggestion
    "utils": "@/lib/shadcn/utils",
```